internal/validation/validation.go

```diff
@@ -45,3 +45,24 @@ func FileExists(filename string) bool {
 	}
 	return !info.IsDir()
 }
+
+func IsSubDirValid(base, subdir string) error {
+	if prefixExists(subdir) {
+		return fmt.Errorf(`project subdirectory should not start with prefix: %s`, string(subdir[0]))
+	}
+	path := filepath.Join(base, subdir)
+	pathInfo, err := os.Stat(path)
+	if err != nil {
+		return err
+	}
+
+	if !pathInfo.IsDir() {
+		return fmt.Errorf(`path: %s is not a directory`, path)
+	}
+
+	return nil
+}
+
+func prefixExists(path string) bool {
+	return path[0] == '\\' || path[0] == '/' || path[0] == '.'
```

Comment:
for `\\` would this need to be `return path[0] == '\'` OR `(path[0] == '\' && path[1] == '\')`

Reply:
but the test passes.... what am i missing? 😂

Reply:
I think I could change the delimiter but it wasn't happy with backticks iirc, but it's escaping the `\`

Reply:
AHHHHHHHHHHH thanks for the clarification! i probably would have realized that if i opened it in VSCode 🤦‍♀️